src/lib/server/migrations.ts

```diff
@@ -436,6 +440,37 @@ const migrations = [
 		run: async () => {
 			await collections.pictures.dropIndex('productId_1').catch(console.error);
 		}
+	},
+	{
+		_id: new ObjectId('39811201e92e590e858af8bb'),
```

Comment:
This is an id from the year `2000`. Use `new ObjectId()` locally to generate a new one